configs/wip-suite.yaml

```diff
@@ -0,0 +1,3 @@
+tests:
+  testsToRun:
+    - '[Suite: wip]'
```

Comment:
This should be set to `informing` not `wip`

Reply:
So, I took wip to be useful as I don't think this test is actually ready to run yet due to its dependencies. @2uasimojo , is that right?

Reply:
All of the dependencies have been merged. :) I checked before doing my review.

Reply:
Well snap. :-) Nevermind then!

Reply:
I was careful to leave myself an "out" with "and more" :P

See main thread for more...